pom.xml

```diff
@@ -339,6 +339,25 @@
         </plugins>
       </reporting>
     </profile>
+    <profile>
```

Comment:
This looks good. Can you also update BeeJU please:
- https://github.com/HotelsDotCom/beeju/blob/master/pom.xml#L117
- https://github.com/HotelsDotCom/beeju/blob/master/.travis.yml#L12